modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -24,6 +24,8 @@ If you're using self-managed Kubernetes on AKS and have the AKS-managed Azure AD
 `az aks get-credentials --name <NAME> --resource-group <RESOURCE_GROUP> --admin` 
 ====
 
+For Runtime Fabric on Self-managed Kubernetes:
+
 . Download the `rtfctl` command-line utility:
 +
 `rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
```

Comment:
You may want to use "following" instead of "below". I think "below" is not accessible.